notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb

```diff
@@ -137,8 +137,10 @@
    },
    "outputs": [],
    "source": [
-    "from mmlspark import FindBestModel\n",
-    "bestModel = FindBestModel(evaluationMetric=\"AUC\", models=lrmodels).fit(test)"
+    "from mmlspark import FindBestModel, BestModel\n",
+    "bestModel = FindBestModel(evaluationMetric=\"AUC\", models=lrmodels).fit(test)\n",
+    "bestModel.write().overwrite().save('model.mml')\n",
+    "loadedBestModel = BestModel.load('model.mml')"
```

Comment:
Oh I see, ``FindBestModel`` used to return ``FindBestModelModel`` but now returns just ``BestModel``?